### DISCUSSION TITLE
JSON Patch web API

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1082] [Q&A]
My current project requires me to update a single field on the database.  Currently Oqtane has no pattern for this, that I am aware of so my question is - Will the framework support this or should I go down my own route?

**Comment by sbwalker** [comment]
@leigh-pointer I am not sure I understand your question... Oqtane uses EF Core so it uses the ORM approach for interacting with the database. For updates this means that you retrieve an object, change a property value on the object, and then call update and it will change that field in the database.

**Reply by leigh-pointer** [reply]
@sbwalker  Im wanting to update one field on the record without retrieving the whole object.  It seems a big overhead to retrieve an object to just update a flag or a count. [jsonpatch aspnetcore-5.0](https://docs.microsoft.com/en-us/aspnet/core/web-api/jsonpatch?view=aspnetcore-5.0)

**Reply by sbwalker** [reply]
@leigh-pointer you would be better solving this within your own module for now, as the changes to support this feature consistently across the Oqtane API would be very substantial, and there is currently no use case in the framework which would actually utilize it.
